chia/cmds/farm_funcs.py

```diff
@@ -232,6 +233,10 @@ async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int,
     minutes = -1
     if blockchain_state is not None and plots is not None:
         proportion = total_plot_size / blockchain_state["space"] if blockchain_state["space"] else -1
-        minutes = int((await get_average_block_time(rpc_port) / 60) / proportion) if proportion else -1
+        if 1 / proportion > BLOCKS_PER_DAY:
+            average_block_time = SECONDS_PER_BLOCK
+        else:
+            average_block_time = await get_average_block_time(rpc_port)
+        minutes = int((average_block_time / 60) / proportion) if proportion else -1
     print("Expected time to win: " + format_minutes(minutes))
```

Comment:
There's a lot of double inversions going on. I think those lean mean the same as
```python
space = blockchain_state["space"]
if blockchain_state is None or plots is None or not space:
    print(f"Expected time to win could not be computed.")
else:
    multiple = int(space / total_plot_size)  # Inverse of the portion of the network space
    block_time = (await get_average_block_time(rpc_port)) if multiple < BLOCKS_PER_DAY else SECONDS_PER_BLOCK
    print(f"Expected time to win: {format_minutes(multiple * int(block_time / 60))}.")
```
----
Also, holy why is `format_minutes` 60 lines long and only takes ints?

Reply:
this seems unrelated to this pr

Reply:
How so? I mean it's the suggestion to not do several double divisions. But it's also just a suggestion of course.

Reply:
I was responding to the comment about format_minutes being 60 lines long, I will defer to someone with merge privs if refactoring is desired before this PR is merged (it could obviously happen after, too, and I prefer to keep my PRs as small and focused as possible)